mysql/datadog_checks/mysql/statement_samples.py

```diff
@@ -720,7 +723,7 @@ def _collect_statement_samples(self):
 
         start_time = time.time()
 
-        rows = self._get_new_events_statements(events_statements_table, self._events_statements_row_limit)
+        rows = self._get_new_events_statements_current()
         rows = self._filter_valid_statement_rows(rows)
         events = self._collect_plans_for_statements(rows)
```

Comment:
I think it could be wise to update the explain rate limiter interval: https://github.com/DataDog/integrations-core/blob/000efc86d1cfd5b60a1b203abcafd6b9522ef065/mysql/datadog_checks/mysql/statement_samples.py#L383

Since we are seeing more unique executions of a query, there would be more advantage to explaining more queries with potentially different parameters (thus generating more unique normalized plans).

Reply:
I increased `ttl` by 24%. In doing so, the amount work should remain approximately the same as, based on the test with the order app, we're capturing 24% less samples.

Reply:
You **increased** the TTL? I thought we would want the opposite? (decrease the TTL to capture queries 24% more frequently)

Reply:
I misunderstood how the cache limits the rate. Since `explain` executes only if the query isn't in cache, smaller TTL will cause more frequent `explain` executions. I decreased the TTL value accordingly.